texasbbq.py

```diff
@@ -218,13 +321,12 @@ def clone_url(self):
 
     @property
     def git_ref(self):
-        """ The target git ref to checkout.
+        """The target git ref to checkout.
 
         This function must work out which ref (branch or tag should be checked
         out and return that. A good start is to use
         `git_ls_remote_tags(self.clone_url)` to obtain a list of tags from the
         remote. If you want to use `master` just return that. If you specify
-        `clone_url` you should also specify this.
```

Comment:
why delete this?

Reply:
Because the class has changed to no longer support this. You must supply a `clone_url` now. The docs are now just following suit.

Reply:
Perhaps delete the start of the sentence too  ? :)

Reply:
Yup, just saw that and did it.